src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Compile.cs

```diff
@@ -94,7 +94,7 @@ protected string GenerateCommandLineCommands (ITaskItem dir, string outputArchiv
 			cmd.AppendSwitchIfNotNull ("-o ", outputArchive);
 			if (!string.IsNullOrEmpty (ResourceSymbolsTextFile))
 				cmd.AppendSwitchIfNotNull ("--output-text-symbols ", ResourceSymbolsTextFile);
-			cmd.AppendSwitchIfNotNull ("--dir ", ResourceDirectoryFullPath (dir.ItemSpec));
+			cmd.AppendSwitchIfNotNull ("--dir ", dir.ItemSpec.TrimEnd ("\\"));
```

Comment:
Turns out `ResourceDirectoryFullPath` is used to fix up the error messages for VS. So lets try to not full path the directory and just `TrimEnd`.